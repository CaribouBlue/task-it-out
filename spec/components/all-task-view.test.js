import React from 'react';
import renderer from 'react-test-renderer';
import AllTaskView from 'src/components/all-task-view';
import MockTasks from 'mock-data/tasks';

let consoleErrorSpy = null
beforeEach(() => {
  consoleErrorSpy = jest.spyOn(global.console, 'error')
})

afterEach(() => {
  expect(consoleErrorSpy).not.toHaveBeenCalled();
})

test('AllTaskView component matches snapshot', () => {
  const component = renderer.create(
    <AllTaskView tasks={MockTasks}></AllTaskView>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
