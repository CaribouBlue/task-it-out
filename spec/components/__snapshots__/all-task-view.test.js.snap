// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllTaskView component matches snapshot 1`] = `
<table>
  <tbody>
    <tr>
      <td>
        task1
      </td>
      <td>
        Sat Sep 19 2020
      </td>
      <td>
        open
      </td>
    </tr>
    <tr>
      <td>
        task2
      </td>
      <td>
        Sat Sep 19 2020
      </td>
      <td>
        open
      </td>
    </tr>
    <tr>
      <td>
        task3
      </td>
      <td>
        Sat Sep 19 2020
      </td>
      <td>
        open
      </td>
    </tr>
  </tbody>
</table>
`;
